extract_wheels/lib/namespace_pkgs_test.py

```diff
@@ -133,17 +131,5 @@ def test_empty_case(self):
         self.assertEqual(actual, set())
 
 
-def main():
```

Comment:
We didn't actually need this?

Reply:
We did when the tests were in a folder, but not since they are now per-file.

Reply:
Oh yeah that makes sense.